models/commons/src/high-precision-money/generator.ts

```diff
@@ -0,0 +1,16 @@
+import { fake, Generator, oneOf } from '@commercetools-test-data/core';
+import { THighPrecisionMoney } from './types';
+
+// https://docs.commercetools.com/api/types#highprecisionmoney
+
+const generator = Generator<THighPrecisionMoney>({
+  fields: {
+    type: 'highPrecision',
+    centAmount: fake((f) => f.number.int({ min: 1, max: 999999 })),
```

Comment:
I think it would be more coherent if we were to compute centAmount from the generated preciseAmount and fractionDigits. ['postBuild' on the generator](https://github.com/jackfranklin/test-data-bot/blob/1dd9bcc0c10a531f9f7379a7e70bb4c318610d26/README.md?plain=1#L280) should allow that.